packages/flutter_tools/gradle/build.gradle.kts

```diff
@@ -17,6 +17,11 @@ gradlePlugin {
             id = "dev.flutter.flutter-gradle-plugin"
             implementationClass = "FlutterPlugin"
         }
+        // The "flutterPluginPlugin" name isn't used anywhere.
```

Comment:
I copied this comment style, but also open to just removing - I don't think noting that the names we choose for every plugin aren't used elsewhere is helpful.

Reply:
it might be more helpful to have a list of all our names that the comment references instead.

Reply:
Resolving because we aren't making a plugin any more